apps/asana/src/inngest/functions/users/schedule-token-refresh.ts

```diff
@@ -0,0 +1,38 @@
+import { gte } from 'drizzle-orm';
+import { env } from '@/env';
+import { db } from '@/database/client';
+import { Organisation } from '@/database/schema';
+import { inngest } from '../../client';
+
+export const scheduleTokenRefresh = inngest.createFunction(
+  { id: 'schedule-token-refresh' },
+  { cron: env.TOKEN_REFRESH_CRON },
+  async ({ step }) => {
+
+    const thirtyMinutesAgo = new Date(Date.now() - 30 * 60000);
```

Comment:
We should select token that will expires in thirty minutes
```suggestion
    const thirtyMinutesFromNow = new Date(Date.now() + 30 * 60000);
```